tfjs-backend-wasm/src/cc/BUILD

```diff
@@ -37,6 +37,32 @@ cc_binary(
     ],
 )
 
+cc_binary(
+    name = "tfjs-backend-wasm-threaded-simd.js",
+    srcs = ["backend.cc"] + KERNELS_WITH_KEEPALIVE,
+    linkopts = [
+        "-s ALLOW_MEMORY_GROWTH=1",
+        "-s DEFAULT_LIBRARY_FUNCS_TO_INCLUDE=[]",
+        "-s DISABLE_EXCEPTION_CATCHING=1",
+        "-s FILESYSTEM=0",
+        "-s EXIT_RUNTIME=0",
+        "-s EXPORTED_FUNCTIONS='[\"_malloc\", \"_free\"]'",
+        "-s EXTRA_EXPORTED_RUNTIME_METHODS='[\"cwrap\"]'",
+        "-s MODULARIZE=1",
+        "-s EXPORT_NAME=WasmBackendModuleThreadedSimd",
+        "-s MALLOC=emmalloc",
+        "-s USE_PTHREADS=1",
+        "-s PROXY_TO_PTHREAD=1",
+        # Many x86-64 processors have 2 threads per core, so we divide by 2.
+        "-s PTHREAD_POOL_SIZE=" +
+        "'Math.min(4, Math.max(1, (navigator.hardwareConcurrency || 1) / 2))'",
```

Comment:
Is it doable that we register a flag `WASM_PTHREAD_POOL_SIZE` in `flags_wasm.ts` and initialized with `'Math.min(4, Math.max(1, (navigator.hardwareConcurrency || 1) / 2))'`. Then in here and `backend.cc`, we both use the flag value?